tests/checker/test_news.py

```diff
@@ -25,11 +25,13 @@
 # to enable the has_newsserver() resource manually.
 NNTP_SERVER = "news.uni-stuttgart.de"
 # info string returned by news server
-NNTP_INFO = "200 news.uni-stuttgart.de InterNetNews NNRP server " \
-            "INN 2.5.2 ready (no posting)"
+NNTP_INFO = (
+    "200 news.uni-stuttgart.de InterNetNews NNRP server " "INN 2.5.2 ready (no posting)"
```

Comment:
```suggestion
    "200 news.uni-stuttgart.de InterNetNews NNRP server INN 2.5.2 ready (no posting)"
```

If this is not a Black bug, it should be!

Reply:
https://github.com/psf/black/issues/26

Fixed 21 days ago.

Last release in 2019.

*sigh*

Reply:
Done.